packages/app-store/EventTypeAppContext.tsx

```diff
@@ -5,13 +5,20 @@ export type GetAppData = (key: string) => unknown;
 export type SetAppData = (key: string, value: unknown) => void;
 type LockedIcon = JSX.Element | false | undefined;
 type Disabled = boolean | undefined;
+
+type AppContext = {
+  getAppData: GetAppData;
+  setAppData: SetAppData;
+  LockedIcon: LockedIcon;
+  disabled: Disabled;
+};
 // eslint-disable-next-line @typescript-eslint/no-empty-function
-const EventTypeAppContext = React.createContext<[GetAppData, SetAppData, LockedIcon, Disabled]>([
-  () => ({}),
-  () => ({}),
-  undefined,
-  undefined,
-]);
+const EventTypeAppContext = React.createContext<AppContext>({
+  getAppData: () => ({}),
+  setAppData: () => ({}),
+  LockedIcon: undefined,
+  disabled: undefined,
+});
```

Comment:
I moved the context from an array into an object. Not every item was needed from the context, so with an object, we're able to call on only the things we needed rather than everything.

https://github.com/calcom/cal.com/pull/9950#discussion_r1253489346